index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home | Julie Warren</title>
+  <link rel="stylesheet" href="style.css" />
+  <!-- favicon -->
+  <link rel="apple-touch-icon" sizes="180x180" href="./assets/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./assets/favicon/favicon-16x16.png">
+  <link rel="manifest" href="./assets/favicon/site.webmanifest">
+  <link rel="mask-icon" href="./assets/favicon/safari-pinned-tab.svg" color="#087e8b">
+  <meta name="msapplication-TileColor" content="#2b5797">
+  <meta name="theme-color" content="#ffffff">
+</head>
+<body class="home-page">
+  <!-- navigation bar -->
+  <header class="home-header">
+    <nav class="nav-container">
+      <a href="index.html">
+        <img id="nav-logo" alt="logo of initials J A W enclosed in a circle" src="./assets/images/JAW.png">
+      </a>
+      <ul class="nav-buttons">
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="pages/about.html">About</a>
+        </li>
+        <li>
+          <a href="pages/portfolio.html">Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <!-- main body -->
+  <main class="main-container-home">
+    <div class="left-side-container">
+      <h1>
+        Julie Anne Warren
+      </h1>
+      <h2>
+        MA → WA
+        <br>
+        Rowing Coach → Full Stack Software Engineer
+      </h2>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)